src/components/UI/Alert/ImageUploaderDialog.vue

```diff
@@ -193,7 +188,6 @@ export default {
       // save the image info locally
       // extract the base64 URL from the info and
       // use it to show the preview
-      this.startLoading();
```

Comment:
why move this inside VueImageUploader? That is like changing an imported component with our custom code right? can we avoid it?